yarn.lock

```diff
@@ -1627,6 +1686,13 @@ __metadata:
   languageName: node
   linkType: hard
 
+"@endo/env-options@npm:^1.1.7":
+  version: 1.1.7
+  resolution: "@endo/env-options@npm:1.1.7"
+  checksum: 10c0/5784bd68790041b08d9ead4f6c29cc7871d2e554c23fc44fff38cb20b6b4e55cdba2f78d844ba5ad4b0818185c32475ff318c1b77890d628690d7c7a6ede9475
+  languageName: node
+  linkType: hard
+
 "@endo/errors@npm:^1.2.4":
```

Comment:
This approach doesn't seem correct since we now have two versions of the endo packages.

They seem to trace to back to the `community-dev` branch. What's the best way to approach this?